### ISSUE TITLE
ambiguité de traduction plurielle  : src/main/Rdynload.c:1323

### ISSUE THREAD

**Issue by ChristianWia** [phgrosjean/rfrench#24]
#: src/main/Rdynload.c:1323
#, c-format
msgid "cannot allocate memory for registered native symbol (%d byte)"
msgid_plural "cannot allocate memory for registered native symbol (%d bytes)"
msgstr[0] ""
"impossible d'allouer de la mémoire pour le symbole natif enregistré (%d byte)"
msgstr[1] ""
"impossible d'allouer de la mémoire pour les symboles natifs enregistrés (%d "
"bytes)"

Je suis un peu perdu: sur quoi porte la forme plurielle svp ?
1. si c'est sur le nombre de symboles, alors : 
 - la forme EN doit etre  'registered native symbol **S** ' et il faut demander à corriger le source sur le serveur
 - la forme FR est bonne
2. si c'est sur le nombre d'octets occupés par le symbole
 - la forme EN est bonne
 - il faut corriger le FR pluriel en ' pour le symbole natif enregistré'

Merci.

**Comment by phgrosjean** [comment]
@ChristianWia Non, ici, le pluriel porte uniquement sur le nombre d'octets: singulier = 1 byte, pluriel = several bytes, sinon, on aurait "symbols" au pluriel en anglais. Donc:

```
"impossible d'allouer de la mémoire pour le symbole natif enregistré (%d octet)"
"impossible d'allouer de la mémoire pour le symbole natif enregistré (%d octets)"
```

**Comment by ChristianWia** [comment]
justement, ils ont peut être oublié le 's' dans la forme anglaise - d'où ma question. Mais j'ai corrigé comme proposé ci-dessus cad avec 'le symbole'.
